assets/vue/Doughnut.vue

```diff
@@ -0,0 +1,61 @@
+<template>
+    <Doughnut :chart-data="chartData" :chart-options="chartOptions" />
+</template>
+
+<script lang="ts" setup>
+    import { Doughnut } from "vue-chartjs";
+    import { Chart as ChartJS, registerables } from "chart.js";
+    ChartJS.register(...registerables);
+
+    interface DataPoint {
+        name: string,
+        count: number,
+    }
+    interface Props {
+        doughnutData: DataPoint[],
+    }
+    const props = withDefaults(defineProps<Props>(), {
+        doughnutData: () => [{ name: "Missing Data", count: 1 }],
+    });
+
+    const { doughnutData } = toRefs(props);
```

Comment:
toRefs kommt ja von vue. gibt es dass auch in nuxt? (wie dass man ref mit useState ersetzen sollte)

Reply:
Nein, not that I know of.